app/experimenter/nimbus-ui/src/components/PageRequestReview/index.test.tsx

```diff
@@ -44,16 +42,13 @@ describe("PageRequestReview", () => {
   it("renders as expected", async () => {
     const { mock } = mockExperimentQuery("demo-slug");
     render(<Subject mocks={[mock]} />);
-    let submitButton: HTMLButtonElement;
     await waitFor(() => {
       expect(screen.getByTestId("PageRequestReview")).toBeInTheDocument();
-      submitButton = screen.getByText("Launch") as HTMLButtonElement;
+      expect(
+        screen.getByTestId("start-launch-draft-to-review") as HTMLButtonElement,
+      ).toBeInTheDocument();
```

Comment:
Optional: I recently discovered that we never need to use `expect(...).toBeInTheDocument()` because testing-library has an async method for this, so:

```tsx
// this
expect(screen.getByTestId("start-launch-draft-to-review") as HTMLButtonElement).toBeInTheDocument();

// becomes this
await screen.findByTestId('start-launch-draft-to-review');
```

Slightly less bulky, and if you do it for both we can eliminate the `waitFor` entirely here. But again optional - I plan to do a therapeutic sweep for these at some point.

(Also, you shouldn't need to cast to `HTMLButtonElement` if we're just asserting the element's presence.)